cob_hardware_emulation/scripts/emulation_follow_joint_trajectory.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from past.utils import old_div
```

Comment:
Maybe this needs to be installed/added to deps or just undo the change related to old_dev below as well.

Reply:
done